public/js/components/Scopes.js

```diff
@@ -39,7 +39,7 @@ function getSpecialVariables(pauseInfo, parentName) {
     vars.push({
       name: "<exception>",
       path: parentName + "/<exception>",
-      contents: { value: thrown.toJS() }
+      contents: { value: thrown }
```

Comment:
Not sure what I'm supposed to look at there? :) Can you just reproduce it locally and debug it from there?